acceptance/acceptance_test.go

```diff
@@ -476,6 +476,7 @@ func testWithoutSpecificBuilderRequirement(
 			output, err := pack.Run(
 				"build", "some/image",
 				"-p", filepath.Join("testdata", "mock_app"),
+				"--default-process", "web",
```

Comment:
Adding of `"--default-process", "web",` is a temporary work around for an know lifecycle issue: https://github.com/buildpacks/lifecycle/issues/376